users/config.rst

```diff
@@ -591,6 +591,10 @@ Listen Addresses
 
 The following address types are accepted in sync protocol listen addresses:
 
+Default listen addresses (``default``)
+    This is equivalent to ``tcp://0.0.0.0:22000`` and
```

Comment:
The `listenAddress` explanation should state that the element can be present any number of times, but that's sort of unrelated to this improvement.